lib/ansible/modules/cloud/amazon/ec2_elb_facts.py

```diff
@@ -80,6 +80,9 @@
 
 import traceback
 
+from ansible.module_utils.basic import *
```

Comment:
Please import what you need rather than `*`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#python-imports